zipline/pipeline/factors/__init__.py

```diff
@@ -62,4 +65,7 @@
     'TrueRange',
     'VWAP',
     'WeightedAverageValue',
+    'MovingAverageConvergenceDivergence',
```

Comment:
The rest of the entries here were in sorted order prior to these additions.  It's nice for a reader looking for an entry to maintain that order.